src/components/HomePage/SearchArea/jobTypes.js

```diff
@@ -0,0 +1,25 @@
+const JOB_TYPES = Object.freeze([
+    {
+        value: "ANY",
+        label: "Any",
+    },
+    {
+        value: "SUMMER_INTERNSHIP",
+        label: "Summer Internship",
+    },
+    {
+        value: "CURRICULAR_INTERNSHIP",
+        label: "Curricular Internship",
+    },
+    {
+        value: "PART_TIME",
+        label: "Part-Time",
+    },
+    {
+        value: "FULL_TIME",
+        label: "Full-Time",
+    },
+
+]);
+
```

Comment:
I don't think we should have an `OTHER` option as a filter, as it provides no information. Instead, we should only have concrete filters, and update them as we see fit/necessary over time.
Changing the value of ANY to `undefined`

Reply:
However, this disables the ability to search for something that is not specified in the current filter (it would only be searchable using `ANY`, which would show everything else). I don't think that this is very good but using `Other` would also be less than ideal, as it would be manual database migrating if other possibilities are added 🙃

Reply:
I think the idea of having all the possibilities disclosed is the best. It removes ambiguity and allows for better searching. Regarding searching for something that is not specified in the current filter, it would not happen because everything is part of those options and, as I mentioned before, I can always add more options, but I don't think there are many more, so it would be a good trade-off

Reply:
Maybe not in the case for job types, but maybe in other things, I guess

Reply:
Yeah, there may be filters that need an `OTHER` value, but I think jobTypes is not one of them